packages/nimble-components/src/table-column/date-text/index.ts

```diff
@@ -10,7 +11,34 @@ import type { ColumnInternalsOptions } from '../base/models/column-internals';
 
 export type TableColumnDateTextCellRecord = TableNumberField<'value'>;
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
-export interface TableColumnDateTextColumnConfig {}
+export interface TableColumnDateTextColumnConfig {
+    format?: 'custom';
```

Comment:
Was there a specific rational for hard-coding the possible strings rather than following the existing pattern in nimble of creating enums? Part of the reason I ask is because it looks like these values are duplicated in a few different places.

Reply:
It looked like it was going to be a bunch of enums that wouldn't get reused much, but on second look, I can see benefit to creating them.